src/components/NativePicker/pickerStyles/index.android.js

```diff
@@ -1,9 +1,9 @@
 import styles from '../../../styles/styles';
 
 const pickerStyles = (disabled, error, focused) => ({
-    ...styles.expensiPicker(disabled, error, focused),
+    ...styles.Picker(disabled, error, focused),
```

Comment:
I understand that `ctrl-shift+F` works great. But please change it to camelCase and other instances if any.